app/scripts/lib/validate.js

```diff
@@ -103,7 +103,41 @@ define([
       }
 
       return password.length >= Constants.PASSWORD_MIN_LENGTH;
+    },
+
+    /**
+     * Check if a value is not falsy.
+     *
+     * @return the value if it's not falsy, the defaultValue otw.
+     */
+    isNotFalsy: function (value, defaultValue) {
+      return value || defaultValue;
+    },
+
+    /**
+     * Check if at least one of the arguments is not falsy.
+     *
+     * @return the first non falsy argument if there is one, the last argument
+     * otw.
+     */
+    areNotFalsy: function () {
```

Comment:
My brain has a hard time with `areNotFalsy` and `isNotFalsy`. Double negatives cause me to have to do the negation in my head whereas a function name like `firstTruthy` is easier to reason about with no additional thought.

Which reminds me, this function and `isNotFalsy` are equivalent in the two argument case. Might as well collapse them.